stages/cloud-provider/aws/lead/variables.tf

```diff
@@ -101,4 +101,11 @@ locals {
 
 variable "docker_registry_mirror" {
   default = ""
+}
+
+variable "velero_user" {
+  default = "velero"
+}
+
+variable "enable_velero" {
```

Comment:
I think we should provide a default for this.  probably `false`

Reply:
Agreed, will update.